### ISSUE TITLE
Force browser to clear cache after upgrade

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#439]
When we upgrade (or hotfix) the server, users often have trouble with cached files. There are methods to force that from the server, like appending a GET parameter to .js and .css requests with the version number, like: `ext.min.js?v=2.18`.

The method is described here: https://stackoverflow.com/a/1922924

**Comment by jaragunde** [comment]
Apparently, a variation of this technique is still used, it's called "cache busting": https://javascript.plainenglish.io/what-is-cache-busting-55366b3ac022

**Comment by jaragunde** [comment]
Fixed with these commits, reviewed in PR #584 and #586:
```
commit c860dc1ff8bcd383c036365bea40e3fbf8f85b57 
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Mon May 9 10:23:50 2022 +0200

    [#439] Properly minify holidayManagement.js.
    
    We also tune a bit the `find` commands, to reduce the search scope.

commit 98e81bfd974b0fec21fed43a589128f788b86086
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Mon May 2 13:01:09 2022 +0200

    [#439] Append a hash to minified file names.
    
    This will enforce browsers to retrieve again the file when the hash
    changes, making sure users don't use old versions of the JS code.
    
    The minify script is changed in several ways, in addition to
    calculating and appending the hash to the file name:
    
    * To revert previous minification, we now use git reset --hard. It's
      dangerous but prevents the regexp logic from getting too wild.
    
    * We now loop through the php files for every js file, making the
      script much less efficient. Again, this is done to simplify regexp
      logic.
    
    * Explicitly delete source map files from previous minification, to
      prevent maps with different hashes to pile up.

```
